src/scancode/cli.py

```diff
@@ -131,6 +131,29 @@ def print_about(ctx, param, value):
 
     scancode -f json -l -c samples/zlib/ > scan.json
 
+Scan a directory while ignoring a single file. Print scan results to stdout as JSON:
```

Comment:
One question, and this may need to be explained here and you may need a test for it: is the README ignored at the root of the project only? or anywhere in the scanned tree? and can I use a path? e.g. is the glob applied to the path as a string? or to the filenames only?